query-connector/e2e/customize_query.spec.ts

```diff
@@ -0,0 +1,193 @@
+// @ts-check
+
+import { test, expect } from "@playwright/test";
+import { TEST_URL } from "../playwright-setup";
+import { PAGE_TITLES } from "@/app/query/stepIndicator/StepIndicator";
+
+import { TEST_PATIENT, TEST_PATIENT_NAME } from "./constants";
```

Comment:
I'm not sure when this got added but definitely makes it easier to write tests!

Reply:
I thought the same thing! Was so glad to see it just appear haha. Might be harder to use once we support multiple default or test patients like we used to, but for now it's absolutely great.